Tests/Resources/IdP/02-introspect-response.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app": {
+        "type": "object",
+        "value": {
+            "id": "Ooa5cdrlgjv2B21m53w6",
+            "label": "Facebook idp SPA",
+            "name": "oidc_client"
+        }
+    },
+    "cancel": {
+        "accepts": "application/json; okta-version=1.0.0",
+        "href": "https://idx-devex.clouditude.com/idp/idx/cancel",
+        "method": "POST",
+        "name": "cancel",
+        "produces": "application/ion+json; okta-version=1.0.0",
+        "rel": [
+            "create-form"
+        ],
+        "value": [
+            {
+                "mutable": false,
+                "name": "stateHandle",
+                "required": true,
+                "value": "ahc52KautBHCANs3ScZjLfRcxFjP_N5mqOTYouqHFP",
```

Comment:
Can you make sure you update the JSON samples to redact IDs, and the domain for the examples?  e.g. `example.com` for the host name, and altered client and IDP IDs?

Reply:
IDs, tokens etc. are dummy.
I missed domains :( Now it's fixed.